app/components/Exchange/Exchange.jsx

```diff
@@ -2064,97 +2050,142 @@ class Exchange extends React.Component {
+                        ])}
+                        onFlip={
+                            !flipBuySell ? this._flipBuySell.bind(this) : null
+                        }
+                        onTogglePosition={
+                            this.state.buySellTop && !verticalOrderBook
+                                ? this._toggleBuySellPosition.bind(this)
+                                : null
+                        }
+                        moveOrderForm={
+                            !smallScreen && (!flipBuySell || verticalOrderForm)
+                                ? this._moveOrderForm.bind(this)
+                                : null
+                        }
+                        verticalOrderForm={
+                            !smallScreen ? verticalOrderForm : false
+                        }
+                        isPanelActive={isPanelActive}
+                        activePanels={activePanels}
+                        singleColumnOrderForm={singleColumnOrderForm}
+                        hideFunctionButtons={hideFunctionButtons}
+                    />
+                </Tabs.TabPane>
+                <Tabs.TabPane
+                    tab={counterpart.translate("exchange.scaled")}
+                    key={"scaled"}
+                >
+                    <ScaledOrderTab
+                        currentPrice={lowestAsk.getPrice()}
+                        lastClickedPrice={
+                            this.state.ask && this.state.ask.priceText
+                        }
+                        currentAccount={currentAccount}
+                        createScaledOrder={this._createScaledOrder}
+                        type={"bid"}
+                        quoteAsset={quote}
+                        baseAsset={base}
+                    />
+                </Tabs.TabPane>
+            </Tabs>
         );
 
         let sellForm = isFrozen ? null : tinyScreen &&
         !this.state.mobileKey.includes("buySellTab") ? null : (
-            <BuySell
-                showScaledOrderModal={this.showScaledOrderModal}
-                key={`actionCard_${actionCardIndex++}`}
-                onBorrow={quoteIsBitAsset ? this._borrowQuote.bind(this) : null}
-                onBuy={this._onBuy.bind(this, "ask")}
-                onDeposit={this._onDeposit.bind(this, "ask")}
-                currentAccount={currentAccount}
-                backedCoin={this.props.backedCoins.find(
-                    a => a.symbol === quote.get("symbol")
-                )}
-                currentBridges={
-                    this.props.bridgeCoins.get(quote.get("symbol")) || null
-                }
-                isOpen={this.state.buySellOpen}
-                onToggleOpen={this._toggleOpenBuySell.bind(this)}
-                parentWidth={centerContainerWidth}
-                styles={{padding: 5, paddingRight: mirrorPanels ? 15 : 5}}
+            <Tabs
+                animated={false}
+                defaultActiveKey={"scaled"}
```

Comment:
default should be limit, and persisted with viewsettings onchange like with other tabs